include/RAJA/policy/loop/reduce.hpp

```diff
@@ -0,0 +1,522 @@
+/*!
+ ******************************************************************************
+ *
+ * \File
+ *
+ * \brief   Header file containing RAJA reduction templates for
+ *          sequential execution.
+ *
+ *          These methods should work on any platform.
+ *
+ ******************************************************************************
+ */
+
+#ifndef RAJA_reduce_loop_HPP
```

Comment:
Do we need this file? We should be able to just use the sequential reduction stuff as is, correct?